test/models/inventory_item_test.rb

```diff
@@ -41,5 +41,11 @@
 
 class InventoryItemTest < ActiveSupport::TestCase
   test_model_actions
+
+  test 'method average_cost_amount' do
+    i = inventory_items(:inventory_items_001)
```

Comment:
We hadn't discuss about that, but use of fixtures in models and libs tests is highly deprecated because fixtures data are not always stable. Use generated objects or factories instead.